babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -356,6 +356,18 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testUsingRequestModifierSinglar() {
+    final String sql = "using (foo a)";
```

Comment:
Suggest using a data type here (e.g. INTEGER or STRING) rather than "a". Same for other tests below.